pyproj/database.pyx

```diff
@@ -400,13 +408,15 @@ def get_units_map(str auth_name=None, str category=None, bint allow_deprecated=F
 
     cdef int num_units = 0
     cdef PJ_CONTEXT* context = pyproj_context_create()
-    cdef PROJ_UNIT_INFO** db_unit_list = proj_get_units_from_database(
-        context,
-        c_auth_name,
-        c_category,
-        bool(allow_deprecated),
-        &num_units,
-    )
+    cdef PROJ_UNIT_INFO** db_unit_list = NULL
+    with nogil:
+        db_unit_list = proj_get_units_from_database(
+            context,
+            c_auth_name,
+            c_category,
+            bool(allow_deprecated),
```

Comment:
I am surprised cython didn't complain about using `bool` with `nogil`.

Reply:
I'm assuming it's valid if it did not complain, but I'm not 100% sure.